src/test.h

```diff
@@ -10,15 +15,20 @@
 
 #define UNIQUE_NAME(base) PP_CAT(PP_CAT(base, __LINE__), __COUNTER__)
 
-#define TEST_CASE(name, block) int UNIQUE_NAME(_test_id_) = reg_test((name), [](GameState *game) -> bool block);
+#define _FILE() PP_CAT_II(__FILE__)
+#define _LINE() PP_CAT_II(__LINE__)
+
+#define TEST_CASE(name, block) int UNIQUE_NAME(_test_id_) = reg_test((name), [](GameState *game) -> bool block, _FILE(), _LINE());
```

Comment:
We should make this int static, so it isn't used in another file by accident even.